primitives/README.md

```diff
@@ -0,0 +1,30 @@
+## Installation
+
+These commands must be run *in the root directory*, i.e. the parent of this one.
+
+To set up TensorFlow (one time requirement):
+
+```
+./configure
```

Comment:
it should be renamed to `./configure.sh`.

Reply:
good catch!